packages/entando-apimanager/modules/api/apiManager.js

```diff
@@ -117,8 +117,14 @@ const getRequestParams = (request) => {
       ...request.headers,
     },
   };
+  const isFormData = request.contentType === 'multipart/form-data';
+  if (isFormData) {
+    delete requestParams.headers['Content-Type'];
```

Comment:
why it's deleted and not replaced by 'multipart/form-data'? apart from that, think we can refactor a bit this code

Reply:
refactored. please review @gidesan